ethereum/blockcreation/build.gradle

```diff
@@ -15,6 +15,7 @@ jar {
 dependencies {
   implementation project(':ethereum:core')
   implementation project(':ethereum:eth')
+  implementation project(':ethereum:p2p')
```

Comment:
We shouldn't need to add this dependency.  I'm guessing you need it for TimerUtil but we need to work something out to avoid it.

One possibility is that `PendingTransactions` shouldn't have `TimerUtil` injected, but just exposes the method that it would periodically call. Then as part of creating `PendingTransactions` a periodic timer can be setup to call that method. `PeriodicTransactions` then remains completely separate from the mechanism used to periodically perform the call.